src/Plugin/Block/ContactsDashboardTabs.php

```diff
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\contacts\Plugin\Block\ContactsDashboardTabs.
- */
-
 namespace Drupal\contacts\Plugin\Block;
```

Comment:
There is a coder review commit in this pull request. Probably best to do that separately